src/node_printer_posix.cc

```diff
@@ -132,6 +163,30 @@ namespace
         {
             result_printer->Set(V8_STRING_NEW_UTF8("instance"), V8_STRING_NEW_UTF8(printer->instance));
         }
+
+        v8::Local<v8::Object> ppd_options = V8_VALUE_NEW_DEFAULT_V_0_11_10(Object);
+
+        const char	*filename;		/* PPD filename */
+        ppd_file_t	*ppd;			/* PPD data */
+        ppd_group_t	*group;			/* Current group */
+        int i;
+        if ((filename = cupsGetPPD(printer->name)) != NULL)
+        {
+            if ((ppd = ppdOpenFile(filename)) != NULL)
+            {
+                 ppdMarkDefaults(ppd);
+                 cupsMarkOptions(ppd, printer->num_options, printer->options);
+
+                 for (i = ppd->num_groups, group = ppd->groups; i > 0; i--, group++)
+                 {
+                    populatePpdOptions(ppd_options, ppd, group);
+                 }
+                 ppdClose(ppd);
+            }
+            unlink(filename);
+        }
+        result_printer->Set(V8_STRING_NEW_UTF8("driver_options"), ppd_options);
+
         v8::Local<v8::Object> result_printer_options = V8_VALUE_NEW_DEFAULT_V_0_11_10(Object);
```

Comment:
This information may not be required for most of the time. I think this would be better to be in a separate routine, let say, "getPrinterDriverOptions". or to be activated by parameter.